GYBITG/FirstViewController.swift

```diff
@@ -9,11 +9,21 @@ import UIKit
 
 class FirstViewController: UIViewController {
 
+    // This is the GameStatRepo that is instanitated in the AppDelegate file
+    var gameRepo: GameStatProtocol?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
     }
 
-
+    // Handle the segues to the GameStatHistoryViewController and GalleryViewController
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // The 'forward' method is located in the UIStoryboardSegue file
+        if(segue.identifier == "segueShowGameStatHistory") {
```

Comment:
Avoid String literals in the code as Compiler doesn’t warn for typos in String literals and it can lead to annoying bugs and crashes.  One way is to define an Enum in the UIStoryboardSegue.swift file like
`enum AppSegue : String {
    case segueShowGameStatHistory, segueModalGameStatForm
}`
then where you would use the String literal, reference the Enum case's 'rawValue' property, like
 `AppSegue.segueShowGameStatHistory.rawValue`

Reply:
Got it, I'll make those changes.